internal/curator/storageclass/storageclass_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2019 David Reiss. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package storageclass
+
+import (
+	"testing"
+
+	"github.com/westerndigitalcorporation/blb/internal/core"
+	"github.com/westerndigitalcorporation/blb/internal/curator/durable/state/fb"
+)
+
+var testBlob = fb.GetRootAsBlobF(fb.BuildBlob(&fb.Blob{
+	Tracts: []*fb.Tract{
+		&fb.Tract{
+			Version: 3,
+			Hosts:   []core.TractserverID{5, 6, 7},
+		},
+		&fb.Tract{
+			Version:   3,
+			Rs83Chunk: &core.TractID{8, 3},
+		},
+		&fb.Tract{
+			Version:    3,
+			Rs125Chunk: &core.TractID{12, 5},
+		},
+	},
+}), 0)
+
+func TestRepl(t *testing.T) {
+	repl := Get(core.StorageClassREPLICATED)
+	if repl == nil {
+		t.Fatalf("no storage class for REPLICATED")
+	}
+
+	if repl.ID() != core.StorageClassREPLICATED {
+		t.Error("wrong ID")
+	}
+
+	var tract fb.TractF
+	if testBlob.Tracts(&tract, 0); repl.Has(&tract) != true {
+		t.Errorf("this should be a repl tract")
+	}
+	if testBlob.Tracts(&tract, 1); repl.Has(&tract) != false {
+		t.Errorf("this should not be a repl tract")
+	}
+	if testBlob.Tracts(&tract, 2); repl.Has(&tract) != false {
+		t.Errorf("this should not be a repl tract")
```

Comment:
Is there a way to distinguish some of these duped testing error messages so we know what cases cause failures?

Reply:
The test output should include line numbers, right? I'm pretty sure it does when you run with `-v` at least.

Reply:
Right. That works for me.